packages/typescript-estree/tests/lib/semantic-diagnostics-enabled.test.ts

```diff
@@ -1,21 +1,21 @@
 import { readFileSync } from 'fs';
 import glob from 'glob';
+import path from 'path';
 import * as parser from '../../src';
-import { extname } from 'path';
 import { formatSnapshotName, isJSXFileType } from '../../tools/test-utils';
 
 /**
  * Process all fixtures, we will only snapshot the ones that have semantic errors
  * which are ignored by default parsing logic.
  */
-const FIXTURES_DIR =
-  '../../node_modules/@typescript-eslint/shared-fixtures/fixtures';
+const FIXTURES_DIR = path.join(__dirname, '../../../shared-fixtures/fixtures');
+
```

Comment:
https://github.com/typescript-eslint/typescript-eslint/blob/8938b0c74b3407728f42910d7ce53cb51d6ac461/packages/typescript-estree/tests/ast-alignment/fixtures-to-test.ts#L28-L31